reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -344,8 +334,7 @@ def init(self, traefik):
                     # service-account-token in order to create new Pods.
 
                     components_k8s_token = \
-                        ['reana-server', 'job-controller',
-                         'workflow-controller']
+                        ['reana-server', 'workflow-controller']
```

Comment:
I just created an issue https://github.com/reanahub/reana-workflow-controller/issues/218 to try to clarify why we "do not need" the API token inside RJC, as it has been required until now.